stylus/settings/palette.styl

```diff
@@ -122,64 +113,112 @@ html
+    instead of absolute colors to facilitate theming.
 
-        Styleguide Settings.theme
-    */
+    We try to use a naming schema that is close to what is use in [Material-UI React's theming](https://material-ui.com/customization/palette/#palette-colors).
+    It is possible that we have some discrepancies. If you see something that seems out
+    of the ordinary and where you think we could reduce the difference with MUI, please
+    do not hesitate to [open an issue](https://github.com/cozy/cozy-ui/issues/new).
 
+    Styleguide Settings.theme
+*/
+html, .CozyTheme--normal
     /*
     Primary
 
     Stylus: primaryColor             - #297EF2, CSS: var(--primaryColor)
     Stylus: primaryColorDark         - #0B61D6, CSS: var(--primaryColorDark)
     Stylus: primaryColorLight        - #5C9DF5, CSS: var(--primaryColorLight)
-    Stylus: primaryColorLighter      - #4B93F7, CSS: var(--primaryColorLighter)
     Stylus: primaryColorLightest     - #9FC4FB, CSS: var(--primaryColorLightest)
     Stylus: primaryContrastTextColor - #FFFFFF, CSS: var(--primaryContrastTextColor)
 
     Styleguide Settings.theme.primary
     */
-html, .CozyTheme--normal
     --primaryColor #297EF2
     --primaryColorDark #0B61D6
     --primaryColorLight #5C9DF5
-    --primaryColorLighter #4B93F7 // should not be used
     --primaryColorLightest #9FC4FB
 
+    /*
+    Secondary
+
+    Stylus: secondaryColor   - #fd7461, CSS: var(--secondaryColor)
+    Stylus: secondaryColorDark   - #E3503B, CSS: var(--secondaryColorDark)
+    Stylus: secondaryColorLight   - #ffdeda, CSS: var(--secondaryColorLight)
+    Stylus: secondaryColorLightest   - #FECDC6, CSS: var(--secondaryColorLightest)
+
+    Styleguide Settings.theme.secondary
+    */
     --secondaryColor #fd7461
     --secondaryColorDark #E3503B
     --secondaryColorLight #ffdeda // lighten(dodgerBlue, 24)
-    --secondaryColorLighter #fd7461 //
     --secondaryColorLightest #FECDC6
 
-    --primaryBackgroundLight: var(--zircon)
+    /*
+    Background colors
+
+    Stylus: primaryBackgroundLight -  #F5FAFF, CSS: var(--primaryBackgroundLight)
+    Stylus: paperBackgroundColor -  #FFF, CSS: var(--paperBackgroundColor)
+    Stylus: defaultBackgroundColor -  #F5F6F7, CSS: var(--defaultBackgroundColor)
+
+    Styleguide Settings.theme.background
+    */
+    --primaryBackgroundLight: #F5FAFF
+    --paperBackgroundColor: #FFF
+    --defaultBackgroundColor: #F5F6F7 // paleGrey
+
+    /*
+    Text colors
+
+    Stylus: primaryTextColor -  #32363F, CSS: var(--primaryTextColor)
+    Stylus: secondaryTextColor - #95999D, CSS: var(--secondaryTextColor)
+    Stylus: primaryContrastTextColor -  #FFF, CSS: var(--primaryContrastTextColor)
+    Stylus: secondaryContrastTextColor -  #FFF, CSS(--secondaryContrastTextColor)
 
-    --primaryTextColor var(--charcoalGrey)
-    --secondaryTextColor var(--coolGrey)
+    Styleguide Settings.theme.text
+    */
+    --primaryTextColor #32363F
+    --secondaryTextColor #95999D
```

Comment:
For for those two CSS variables, I think I do not have the same definition in cozy-bootstrap: https://github.com/cozy/cozy-bootstrap/blob/main/src/cozy/_colors.scss#L26-L28

I have a value in RGBA with alpha < 1. Is it MUI that combines the CSS var and an alpha value from somewhere else? If yes, what do you think we should do for cozy-bootstrap? Use another name for the CSS variable?

Reply:
> Is it MUI that combines the CSS var and an alpha value from somewhere else?

Not that I am aware of, but I think it can depend on the component, for example in Alerts where the `color` [is generated](https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Alert/Alert.js#L70). I guess some components could create a text color with lowered opacity.

> If yes, what do you think we should do for cozy-bootstrap? Use another name for the CSS variable?

I am not sure why you have a different value in cozy-bootstrap. Is it a requirement from @joel-costa ? I know that text with a reduced opacity will work better on colored background, for example in alerts. That said, I do not think this warrants a change between cozy-bootstrap and cozy-ui, I would be in favor to stick with a non-alpha color value ?

Reply:
> Is it a requirement from @joel-costa ?

Yes, but it is also what MUI does. When I inspect a text with default settings in MUI documentation, the color is `rgba(0, 0, 0, 0.87)`.

> I know that text with a reduced opacity will work better on colored background, for example in alerts.

It is also the case for `--primaryTextColor` on inverted theme (and dark theme in the future).

> That said, I do not think this warrants a change between cozy-bootstrap and cozy-ui,

I already have some CSS variables for cozy-bootstrap that are not defined by Cozy-UI, like the text color for alerts. MUI adds the alpha channel in JS, but I can't do that with just CSS variables.

> I would be in favor to stick with a non-alpha color value?

Why should we take the non-alpha and not the with-alpha color value?

Reply:
Because I don't want to take any risk at this point. If someone wants to take this task, I have no problem.

Reply:
Is it something we need to track somewhere?